### PR TITLE
🦺[Tech Debt] Add Release Issue template

### PR DESCRIPTION
**Pull request by kcw-grunt**
Added release template

### CODE REVIEW

**Review by JCHeist — APPROVED** [review]
Looks good

**Comment by kcw-grunt** [comment]
Thanks @JasperHeist !

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Goal 
+The process of publishing/launching a new release of **Litewallet** should be consistent and drama-free.  It should also allow the team to catch errors before they occur. This template should make it easy for any iOS developer in the team to be confident the release meets all our criteria.
+
+## Prepare code
+- [ ] 1. Locally create a new branch from `develop` with the pattern `release/v#.#.#` (Third dot should always be 0 which is preserved for [PATCHes](https://semver.org))
+- [ ] 2. Note and write down the changes in the `develop` branch since the last release tag. 
+- [ ] 3. Open a draft release at the [litewallet-ios](https://github.com/litecoin-foundation/litewallet-ios/releases) repo targeting the `main` branch and tag with the pattern: **v#.#.#**
+- [ ] 4. Set the release title (should be the same v#.#.#) in [litewallet-ios](https://github.com/litecoin-foundation/litewallet-ios/releases) and add the release notes...add some flair 🤩🔥😅🍀
+- [ ] 5. Save as Draft **Very important!**
+- [ ] 6. Return the the local branch and do a version bump and build number bump. Commit e.g. : `git commit -m 'version bump'`
```
Comment (mosadialiou) [inline .github/ISSUE_TEMPLATE/release.md]: @kcw-grunt replace `Return the  the` ... -> `Return to the`

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+labels: 'Release'
+assignees: 'kcw-grunt'
+
+---
+
+## Goal 
+The process of publishing/launching a new release of **Litewallet** should be consistent and drama-free.  It should also allow the team to catch errors before they occur. This template should make it easy for any iOS developer in the team to be confident the release meets all our criteria.
+
+## Prepare code
+- [ ] 1. Locally create a new branch from `develop` with the pattern `release/v#.#.#` (Third dot should always be 0 which is preserved for [PATCHes](https://semver.org))
```
Comment (mosadialiou) [inline .github/ISSUE_TEMPLATE/release.md]: @kcw-grunt so let's say we have a release called v2.2.0 and then we work on bug fixes only. According to `semver` the release has to be `release/v2.2.1` meaning `The third dot should always be 0` is always true.  Thoughts?

File: .github/ISSUE_TEMPLATE/release.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+name: 🚀[Release] 
+about: An issue template to make sure the release process is followed
+title: '🚀[Release]'
+labels: 'Release'
+assignees: 'kcw-grunt'
+
+---
+
+## Goal 
+The process of publishing/launching a new release of **Litewallet** should be consistent and drama-free.  It should also allow the team to catch errors before they occur. This template should make it easy for any iOS developer in the team to be confident the release meets all our criteria.
```
Comment (mosadialiou) [inline .github/ISSUE_TEMPLATE/release.md]: @kcw-grunt extra space at the beginning of `It should also allow ...`

**Review by mosadialiou — CHANGES_REQUESTED** [review]

**Comment by kcw-grunt** [comment]
Thanks @mosadialiou

File: .github/ISSUE_TEMPLATE/release.md (hunk 2 above)
Comment (kcw-grunt) [reply .github/ISSUE_TEMPLATE/release.md]: Yes.....technically you are correct. It's just have we ever only done releases with bugfixes in the last 2 years?   I was thinking this template is for releases but let me address that.

**Pull request merged**